src/Cake/project.json

```diff
@@ -1,8 +1,9 @@
 ﻿{
-  "version": "0.16.1-*",
+  "version": "0.16.2-*",
```

Comment:
@devlead doesn't this change need to happen in a number of files?  Not just the top level one?